components/kyma-metrics-collector/pkg/process/metrics.go

```diff
@@ -18,6 +19,7 @@ const (
 	globalAccountLabel = "global_account_id"
 	successLabel       = "success"
 	withOldMetricLabel = "with_old_metric"
+	trackable          = "trackable"
```

Comment:
Please change `trackable` to `trackableLabel`.

Reply:
Done